src/tiled/commandmanager.cpp

```diff
@@ -109,7 +109,7 @@ void CommandManager::updateActions()
 
     mActions.append(mSeparator);
 
-    QAction *mEditCommands = new QAction(this);
+    mEditCommands = new QAction(this);
     mEditCommands->setIcon(
             QIcon(QLatin1String(":/images/24x24/system-run.png")));
     mEditCommands->setText(tr("Edit Commands..."));
```

Comment:
Please add a call to `retranslateUi` to the end of the constructor and remove this line, to avoid having the text twice.